libethereum/Executive.h

```diff
@@ -61,7 +61,9 @@ class StandardTrace
     void setShowMnemonics() { m_showMnemonics = true; }
     void setOptions(DebugOptions _options) { m_options = _options; }
 
-    std::string json(bool _styled = false) const;
+    Json::Value jsonValue() const { return m_trace; }
```

Comment:
I would shorten it to `.json()` but I understand that refactoring was easier this way.

Reply:
Yeah I considered it, then decided `jsonValue` is more explicit here, to show that it's not a string

Reply:
:+1: